docs/collections/documents/tutorials/using-spotcollections.md

```diff
@@ -225,97 +225,141 @@ except Exception as e:
 <TabItem value="js" label="Javascript">
 
 ```js
-  'use strict'
-
-  const jsc8 = require('jsc8');
-
-  // Variables - DB
-  const global_url = "https://gdn.paas.macrometa.io";
-  const region_urls = [
-    "https://gdn-sfo2.prod.macrometa.io",
-    "https://gdn-us-west1.prod.macrometa.io",
-    "https://gdn-nyc1.prod.macrometa.io"
-  ];
-
+const jsc8 = require("jsc8");
+
+// Variables - DB
+const globalUrl = "https://gdn.paas.macrometa.io";
+const geofabric = "_system";
+const spotGfName = "spot";
+const regionUrls = [
+  "gdn-us-west",
+  "gdn-ap-west",
+  "gdn-eu-central"
+];
+
+// Variables
+const collectionName = "accounts";
+const readQueryValue = "FOR account IN accounts RETURN account";
+
+// Variables - Data
+const data = [
+  { firstname: "Peter", lastname: "Parker", City: "NewYork" },
+  { firstname: "Bruce", lastname: "Wayne", City: "Gotham" },
+  { firstname: "Clark", lastname: "Kent", City: "Manhatten" },
+  { firstname: "Ned", lastname: "Stark", City: "Winterfell" },
+  { firstname: "Tywin", lastname: "Lannister", City: "Kings Landing" }
+];
+
+const sleep = (milliseconds) => {
+  return new Promise(resolve => setTimeout(resolve, milliseconds));
+};
+
+async function main () {
   // Crete a authenticated instance with Token / Apikey
-  // const client = new jsc8({url: global_url, token: "XXXX", fabricName: '_system'});
-  // const client = new jsc8({url: global_url, apiKey: "XXXX", fabricName: '_system'});
-  // await console.log("Authentication done!!...");
+  // const client = new jsc8({url: globalUrl, token: "XXXX", fabricName: '_system'});
+  // const client = new jsc8({url: globalUrl, apiKey: "XXXX", fabricName: '_system'});
 
   // Or use Email & Password to Authenticate client instance
-  const client = new jsc8(global_url);
-
+  const client = new jsc8(globalUrl);
+  console.log(`1. Connecting to federation: ${globalUrl}`);
   await client.login("nemo@nautilus.com", "xxxxxx");
-
-  //Variables
-  const collection_name = "accounts";
-
-  const read_query = "FOR account IN accounts RETURN account";
-
-  // Variables - Data
-  const data = [
-    {"firstname": "Peter", "lastname": "Parker", "City": "NewYork"},
-    {"firstname": "Bruce", "lastname": "Wayne", "City": "Gotham"},
-    {"firstname": "Clark", "lastname": "Kent", "City": "Manhatten"},
-    {"firstname": "Ned", "lastname": "Stark", "City": "Winterfell"},
-    {"firstname": "Tywin", "lastname": "Lannister", "City": "Kings Landing"},
-  ];
-
-  async function createCollection() {
-    console.log("\n 2. CREATE_COLLECTION");
-
-    try{
-      console.log(`Creating the collection ${collection_name}...`);
-      const exists_coll = await client.hasCollection(collection_name);
-      if (exists_coll === false) {
-          await client.createCollection(collection_name);
+  console.log("The connection was successful");
+
+  // Step 1.5: Assign a spot region.
```

Comment:
@elof and @mikeridley, should this step be removed or rewritten per the upcoming AG scale changes?

Reply:
I think it's ok to keep it because there are still some customers who may use it even though it's not in PaaS yet